### ISSUE TITLE
PixArt license with commercial projects?

### ISSUE THREAD

**Issue by araleza** [PixArt-alpha/PixArt-sigma#112]
Hello, I wanted to ask a question about using PixArt as part of a commercial product.

I'm not too familiar with the AGPL-3.0 license, and other people might have the same question, especially since SD3 seems to not be in the best state right now, so that's why I'm posting here.

Which of the following statements are true about PixArt?

1. PixArt cannot be used with commercial software at all
2. PixArt can be used with commercial software, but some sort of license payment needs to be made (to where?)
3. PixArt can be used with commercial software, but that commercial software must also become open source
4. PixArt can be used with commercial software, and the rest of the commerical software can remain non-open source. Download links to PixArt must be provided. However it cannot be fine-tuned without also releasing the fine-tuned model (4b: and the training data too?)
5. Some other situation?

I think there will be many businesses that were planning on switching to SD3 that are considering what alternatives are possible, and PixArt looks like it's about to get further quality improvements from the community very soon.

**Comment by ReyJ94** [comment]
![image](https://github.com/PixArt-alpha/PixArt-sigma/assets/169773780/095f5c12-474c-4a8f-898d-54a9617e6760)

**Comment by GavChap** [comment]
https://choosealicense.com/licenses/agpl-3.0/ explains the license in plain English

**Comment by GavChap** [comment]
The model itself is openrail++ M which is the same as SDXLs license here. https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/blob/main/LICENSE.md
